swift/swift.py

```diff
@@ -80,7 +81,23 @@ def strinfo(info: AppInfo) -> str:
     Args:
         info: Dataclass object to convert to a JSON string.
     """
-    return json.dumps(asdict(info))
+    return json.dumps(dataclasses.asdict(info))
+
+
+@dataclasses.dataclass
+class Result:

```

Comment:
I think just `Result` is so ambiguous. How about adding some explanations, such as `SwiftcallRequest`?

If you still don't mind, no problem!

Reply:
It makes sense, I will change it to `SwiftcallResult`!